content/en/real_user_monitoring/browser/monitoring_page_performance.md

```diff
@@ -76,6 +76,30 @@ To account for modern web applications, loading time watches for network request
 
 The RUM SDK automatically monitors frameworks that rely on hash (`#`) navigation. The SDK watches for `HashChangeEvent` and issues a new view. Events coming from an HTML anchor tag which do not affect the current view context are ignored.
 
+## Add your own performance timing
+On top of RUM's default performance timing, you may measure where your application is spending its time with greater flexibility. The `addTiming` API provides you with a simple way to add extra performance timing. For example, you can add a timing when your hero image has appeared:
+
+```html
+<html>
+  <body>
+    <img onload="DD_RUM.addTiming('hero_image')" src="/path/to/img.png" />
+  </body>
+</html>
+```
+
+Or when users first scroll:
+
+```javascript
+document.addEventListener("scroll", function handler(e) {
+    //Remove the event listener so that it only triggers once
+    e.currentTarget.removeEventListener(e.type, handler);
+    DD_RUM.addTiming('first_scroll');
+});
+```
+
+Once the timing is sent, you must [create a measure][15] before graphing it in RUM analytics or in dashboards.
```

Comment:
```suggestion
Once the timing is sent, you must [create a measure][15] before graphing it in RUM analytics or in dashboards. Your performance timings will stored under `@view.custom_timings.[YOUR_TIMING_NAME]`.
```
IMO something like that could be valuable for customers to find where it is.
wdyt?